index.js

```diff
@@ -80,6 +80,7 @@ async function loadModels(Bone, models, opts) {
 
   for (const model of models) {
     model.describe();
+    model.didLoad();
```

Comment:
@JimmyDaddy any suggestions about this name?

Reply:
how about `schema` or `schemaLoad` ?

Reply:
It gets called when model is loaded (synchronized with database). `schemaLoad` might be better suited at `realm.schemaLoaded` I think.

Reply:
But as a developer who use leoric,  this function is used to load my custom settings, `schemaLoaded`  is not easy to understand.

Reply:
agreed on `static initialize()`